community/index.html

```diff
@@ -220,172 +158,77 @@ <h4><a href="https://github.com/malaskowski">Maciej Laskowski</a></h4>
-                
-                <a href="https://github.com/marcinkp">
-                  <img class="svg" src="/img/icons/mark-github.svg" alt="Github Link" />
-                </a>
-                
-              </div>
-            </div>
-          </div>
-          
-          <div class="col-sm-4">
-            <div class="contributor">
-              <div class="contributor-left">
-                <a href="https://github.com/oskarjerzyk"><img class="contributor-avatar" src="https://avatars0.githubusercontent.com/u/36707847?s=400&amp;v=4" alt="Oskar Jerzyk Avatar"></a>
-              </div>
-              <div class="contributor-body">
-                <h4><a href="https://github.com/oskarjerzyk">Oskar Jerzyk</a></h4>
-                
-                <a href="https://github.com/oskarjerzyk">
-                  <img class="svg" src="/img/icons/mark-github.svg" alt="Github Link" />
-                </a>
-                
-              </div>
-            </div>
-          </div>
-          
-          <div class="col-sm-4">
-            <div class="contributor">
-              <div class="contributor-left">
-                <a href="https://github.com/tMaxx"><img class="contributor-avatar" src="https://avatars2.githubusercontent.com/u/3840706?s=400&amp;v=4" alt="Nicolas Avatar"></a>
-              </div>
-              <div class="contributor-body">
-                <h4><a href="https://github.com/tMaxx">Nicolas</a></h4>
-                
-                <a href="https://github.com/tMaxx">
-                  <img class="svg" src="/img/icons/mark-github.svg" alt="Github Link" />
-                </a>
-                
-              </div>
-            </div>
-          </div>
-          
-          <div class="col-sm-4">
-            <div class="contributor">
-              <div class="contributor-left">
-                <a href="https://github.com/toniedzwiedz"><img class="contributor-avatar" src="https://avatars2.githubusercontent.com/u/8336085?v=3&amp;s=80" alt="Tomasz Niedzwiedź Avatar"></a>
-              </div>
-              <div class="contributor-body">
-                <h4><a href="https://github.com/toniedzwiedz">Tomasz Niedzwiedź</a></h4>
-                
-                <a href="https://github.com/toniedzwiedz">
-                  <img class="svg" src="/img/icons/mark-github.svg" alt="Github Link" />
-                </a>
-                
-                <a href="https://twitter.com/toniedzwiedz">
-                  <img class="svg" src="/img/icons/twitter-logo.svg" alt="Twitter Link" />
-                </a>
-                
-              </div>
-            </div>
-          </div>
-          
-        </div>
+      <div class="col-sm-6">
+        <h3>Github Issues</h3>
+        <p>All feature requests and bugs can be filed as issues on
+          <a href="https://github.com/Knotx/knotx/issues">Github</a>.
+          Do not use Github issues to ask questions, post them on the User Group.</p>
```

Comment:
How about enabling GH discussion for questions? I don't think we should support User Group any more...